docs/sphinx/dev_guide/ci.rst

```diff
@@ -12,168 +12,252 @@
 Continuous Integration (CI) Testing
 ************************************
 
-.. important:: * All CI test checks must pass before a pull request can be
-                 merged.
-               * The status (pass/fail and run) for all checks can be viewed by
-                 clicking the appropriate link in the **checks** section of a
-                 GitHub pull request.
+.. important:: * All CI checks must pass before a pull request can be merged.
+               * The status (running and pass/fail) for all checks can be 
+                 viewed by clicking the appropriate link in the **checks** 
+                 section of a GitHub pull request.
 
-The CI tools used by the RAJA project, and which integrate with GitHub are:
+The CI tools used by the RAJA project are:
 
   * **Azure Pipelines** runs builds and tests for Linux, Windows, and MacOS 
     environments using recent versions of various compilers. While we do GPU 
     builds for CUDA, HIP, and SYCL on Azure, RAJA tests are only run for 
-    CPU-only pipelines. See the 
-    `RAJA Azure DevOps <https://dev.azure.com/llnl/RAJA>`_ project to learn 
+    CPU-only pipelines. Docker container images we use for the Linux testing
+    are maintained in the 
+    `RSE Ops Project <https://github.com/rse-ops/docker-images>`_. Please see 
+    the `RAJA Azure DevOps <https://dev.azure.com/llnl/RAJA>`_ project to learn 
     more about our testing there.
 
-  * **Gitlab** instances in the Livermore Computing (LC) Center
-    runs builds and tests in LC resource and compiler environments
-    important to many RAJA user applications. Execution of RAJA CI 
-    pipelines on LC Gitlab resources has restrictions described below. If 
-    you have access to LC resources, you can access additional information about
+  * **GitLab** instances in the Collaboration Zone (CZ) of the Livermore 
+    Computing (LC) Center run builds and tests in LC platform and compiler 
+    environments important to many RAJA user applications. Execution of RAJA CI 
```

Comment:
It took me several tries to read the sentence the intended way, that is:
"run builds and test in (LC (platform & compiler)) environments."
I was reading:
"run builds and test in (LC platform) & (compiler environments)"
which sounded weird.
How about:
```suggestion
    Computing (LC) Center run builds and tests in environments combining LC platforms
    and compilers important to many RAJA user applications. Execution of RAJA CI
```

Reply:
I changed it to "Computing (LC) Center run builds and tests on LC HPC platforms using software stacks (compilers, etc.) important to many RAJA user applications."

Is that more clear?

Reply:
Very clear.